django_mailbox/migrations/0006_mailbox_last_polling.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-08-15 22:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_mailbox', '0005_auto_20160523_2240'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='mailbox',
+            name='last_polling',
+            field=models.DateTimeField(blank=True, null=True, help_text='The time of last successfull polling of message. It is blank for new mailbox or processing email via pipe only.', verbose_name='Last polling'),
```

Comment:
"The time of last successful polling for messages.  It is blank for new mailboxes and is not set for mailboxes that only receive messages via a pipe."